src/App.tsx

```diff
@@ -1,117 +1,259 @@
 import { ThemeProvider } from "@mui/material/styles";
 import CssBaseline from "@mui/material/CssBaseline";
 import theme from "./shared-library/theme";
-import Button from "@mui/material/Button";
-import { Stack, Typography,Grid } from "@mui/material";
+import { Button, Stack, Typography, Grid, Paper } from "@mui/material";
 import "./App.css";
+import { Box } from "@mui/system";
 
```

Comment:
nit : import Box from the same `import { Button, Stack, Typography, Grid, Paper } from "@mui/material"; and not from system